.github/workflows/pr.yml

```diff
@@ -10,16 +10,14 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-java@v1
         with:
-          java-version: "11"
```

Comment:
In another PR you went from '11' to "11" do we definitively know which is correct now?

Reply:
Either single or double quotes are acceptable but double quotes allow variable interpolation and escapes. All that matters here is that it is quoted somehow so it gets interpreted as a string, not an integer.